packages/harpguru-core/src/components/harp-face/harp-face.test.tsx

```diff
@@ -12,12 +12,25 @@ jest.mock('reactn')
 const mockUseGlobal = useGlobal as jest.Mock
 
 test('A component is rendered with fragmented face', () => {
+  // TODO: Should be shared resource
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.